### PR TITLE
ignore export_presets.cfg

### PR DESCRIPTION
**Pull request by Alex2782**
Godot has automatically adapted the 'md' and 'gd' files, it is probably this configuration.

 Bildschirmfoto 2023-11-06 um 23 23 15

When I ported the official demos to version 4.0, I had to activate it because otherwise the PR was not accepted.
There are 2 more gd files that I have not included because I edited them myself, debug outputs.

### CODE REVIEW

**Comment by Alex2782** [comment]
perhaps more files could be set to ignore
https://github.com/godotengine/godot-demo-projects/blob/master/.gitignore

**Comment by MewPurPur** [comment]
I don't like removing whitespace on empty lines, I'll add this in the codestyle guidelines in CONTRIBUTING.md.

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 
 # Imported translations (automatically generated from CSV files)
 *.translation
+export_presets.cfg
```
Comment (MewPurPur) [inline .gitignore:7]: However, I'd like to keep this...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ This is the preferred workflow, but tidiness is not as important as work being d
 
 After submitting your pull request, I (MewPurPur) will review your changes and may provide feedback or request modifications. Be responsive to any comments or suggestions. Once your pull request is approved, it will be merged. Afterward, you can delete your branch from your fork.
 
-## Governance 
+## Governance
```
Comment (MewPurPur) [inline CONTRIBUTING.md:18]: this....

File: src/ui_parts/handles_manager.gd [hunk 3]
```diff
@@ -1,4 +1,4 @@
-## Contours drawing and [Handle]s are managed here. 
+## Contours drawing and [Handle]s are managed here.
```
Comment (MewPurPur) [inline src/ui_parts/handles_manager.gd:1]: And this.

Could you update the PR to remove the other changes?

**Comment by Alex2782** [comment]
Now only `ignore`, later I check the project without "trim trailing whitespace". There are other problems. I had to restart Godot after importing, then some scene files were adjusted.

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**